hooks/prebuild.js

```
@@ -46,12 +46,12 @@ module.exports = function (ctx) {
       '/node_modules/@capacitor-community/camera-preview/android/src/main/java/com/ahm/capacitor/camera/preview/CameraPreview.java',
   };
 
-  // Adding GIT_COMMIT_SHA for sentry
+  // Adding mobile app version for tracking the release in sentry
   var mainPath = path.resolve(process.cwd(), 'src/main.ts');
```

Biome

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)